pom.xml

```diff
@@ -42,6 +42,28 @@
     <saga.version>0.0.1-SNAPSHOT</saga.version>
   </properties>
 
+  <scm>
+    <connection>scm:git:https://github.com/ServiceComb/saga.git</connection>
+    <developerConnection>scm:git:https://github.com/ServiceComb/saga.git</developerConnection>
+    <url>https://github.com/servicecomb/saga</url>
+    <tag>${saga.version}</tag>
+  </scm>
+
+  <distributionManagement>
+    <repository>
+      <id>ossrh</id>
+      <name>release</name>
+      <url>https://oss.sonatype.org/service/local/staging/deploy/maven2/</url>
+      <uniqueVersion>true</uniqueVersion>
+    </repository>
+    <snapshotRepository>
+      <id>ossrh</id>
+      <name>snapshot</name>
+      <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+      <uniqueVersion>true</uniqueVersion>
+    </snapshotRepository>
```

Comment:
we have to add a lot more plugins to make auto release work. please refer to java chassis maven release profile